assets/helpers/externalLinks.js

```diff
@@ -40,6 +41,40 @@ const defaultIntCmp = 'gdnwb_copts_bundles_landing_default';
 const androidAppUrl = 'https://play.google.com/store/apps/details?id=com.guardian';
 const emailPreferencesUrl = `https://profile.${getBaseDomain()}/email-prefs`;
 
+function getWeeklyZuoraCode(period: WeeklyBillingPeriod, countryGroup: CountryGroupId) {
+  const urls: EnumMap<WeeklyBillingPeriod, EnumMap<CountryGroupId, string>> = {
+    sixweek: {
+      GBPCountries: 'weeklydomestic-gwoct18-sixforsix-domestic',
```

Comment:
Can we pull the value here out into a variable and reference that given it's the same in many places?